DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Operations/Receive/HostObjectBuilder.cs

```diff
@@ -105,21 +89,15 @@ CancellationToken cancellationToken
       {
         if (IsGISType(obj))
         {
-          var result = ConvertNativeLayers((Collection)obj, path);
-          convertedGISObjects.Add(result);
-          // NOTE: Dim doesn't really know what is what - is the result.path the id of the obj?
-          // TODO: is the type in here basically a GIS Layer?
-          results.Add(new(Status.SUCCESS, obj, result.path, "GIS Layer"));
+          string nestedLayerPath = $"{string.Join("\\", path)}\\{((Collection)obj).name}";
```

Comment:
Since paths are common thing in ArcGIS, I believe it deserves a URL builder utility for these kind of operations. It will be more testable and readable. TBD

Reply:
let's chat about it, this is a specific case, which will be later using all "\\" to create new nested group layer under the "parent" one, to replicate traversed objects structure in the Map Table of Content